docs/pages/dms/client.mdx

```diff
@@ -4,95 +4,147 @@ description: Learn how to create and configure an XMTP client
 
 # Create and configure an XMTP client
 
-The XMTP message API revolves around a network client that allows retrieving and sending messages to other network participants. A client must be connected to a wallet on startup. The client will request a wallet signature in two cases:
-
-1. To sign the newly generated key bundle. This happens only the very first time when key bundle is not found in storage.
-2. To sign a random salt used to encrypt the key bundle in storage. This happens every time the client is started (including the very first time).
+The XMTP message API revolves around a network client that allows retrieving and sending messages to other network participants. 
 
 ## Create a client
 
-A client is created that requires passing in a connected wallet that implements the Signer interface.
+To create an XMTP client, it must be connected to a wallet on startup. You must pass in a connected wallet that implements the Signer interface. This wallet is required for authentication and encryption purposes.
+
+For more information about wallet signature requests, see [Account signatures](/protocol/signatures).
+
+Clients created by XMTP v3 SDKs also support XMTP v2 messages. At a high-level, XMTP v3 is used to support group chat and XMTP v2 is used to support 1:1 chat.
+
+In the near future, XMTP v3 will support both group and 1:1 chat with Messaging Layer Security (MLS). Have questions or feedback, post to the [XMTP Community Forums](https://community.xmtp.org/c/development/58).
+
+Group chat is currently supported for mobile apps in Kotlin, Swift, React Native, and Node.js.
+
+In the near future, XMTP v3 will also support group chat for the web.
+
+For more information about the work being done to support group chat for the web, see this [SQLite, WebAssembly, and Diesel](https://www.youtube.com/watch?v=qnCeXsqqKcY&t=240s) talk in the XMTP Developer Community Call.
+
+If your mobile app provides group chat, you must explicitly configure your client to enable this functionality by setting the `enableV3` client option to `true`.
+
+:::tip[Best practice: Set enableV3 to true]
 
-[Use client configuration options](#configure-the-client) to change parameters of a client's network connection.
+Regardless of whether your app provides group chat, set the `enableV3` client option to `true` to benefit from other aspects of the upgraded XMTP v3 protocol.
+
+:::
+
+For more information about client options, including how to set your client's network environment, see [client configuration options](#configure-the-client).
 
 :::code-group
 
 ```ts [JavaScript]
 import { Client } from "@xmtp/xmtp-js";
+
 // Create the client with a `Signer` from your application
-const xmtp = await Client.create(signer, { env: "dev" });
+const xmtp = await Client.create(signer, {
+  env: "production",
+  enableV3: true, // Enable group chat support
```

Comment:
Does not support v3 at the moment so probably shouldnt add here.

Reply:
@nplasterer Ah, thank you for this - I took it too far! I thought the guidance to have everyone enableV3 meant that at least the flag was available in all SDKs and everyone should set this value. But what we mean is that all apps built with Kotlin, Swift, RN, and Node should set enableV3 to true. GOT IT - thank you!

Reply:
And the code comment is incorrect 😭. So sorry... I promise I know that group chat is not available for web yet.